src/mahoji/commands/config.ts

```diff
@@ -259,15 +259,17 @@ async function handleCombatOptions(user: KlasaUser, command: 'add' | 'remove' |
 	if (!command || (command && command === 'list')) {
 		// List enabled combat options:
 		const cbOpts = settings.combat_options.map(o => CombatOptionsArray.find(coa => coa!.id === o)!.name);
-		return `Your current combat options are:\n${cbOpts.join('\n')}\n\nTry: \`/config user command_options help\``;
+		return `Your current combat options are:
+		\n${cbOpts.join('\n')}
+		\n\nTry: \`/config user combat_options action:Help\``;
```

Comment:
This is going to cause big empty spaces in the msg. did you test it out?

Reply:
I have sent a new commit which has now fixed this. I did not see the extra line on initial testing, I thought I had used single quotes. I have now changed it to have a different message for `action:List` depending on whether there is any combat_options already enabled